src/Build/Construction/ProjectElementContainer.cs

```diff
@@ -63,19 +64,23 @@ internal ProjectElementContainer(XmlElement xmlElement, ProjectElementContainer
         /// Get an enumerator over all children, gotten recursively.
         /// Walks the children in a depth-first manner.
         /// </summary>
-        public IEnumerable<ProjectElement> AllChildren => GetChildrenRecursively();
+        public IEnumerable<ProjectElement> AllChildren => GetDescendants();
```

Comment:
```suggestion
        public IEnumerable<ProjectElement> Descendants => GetDescendants();
```

Reply:
This is part of the public API

Reply:
Maybe the word "children" could be replaced with "descendants" in the doc comment.